mlflow/utils/rest_utils.py

```diff
@@ -108,6 +108,21 @@ def http_request(
         ) from to
     except requests.exceptions.InvalidURL as iu:
         raise InvalidUrlException(f"Invalid url: {url}") from iu
+    except requests.exceptions.RetryError:
+        # When RetryError happens, make one more request to the server (without retries) and
+        # return the server's response to the caller. Displaying a more detailed error message
+        # is helpful for users to debug issues.
+        return _get_http_response_with_retries(
+            method,
+            url,
+            0,
+            backoff_factor,
```

Comment:
Do we need a backoff factor or can we just instantly resend and capture the failure by disabling the geometric backoff logic (setting the value to 0)?

Reply:
This change is not relevant anymore. Please see the comment below :)